x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/policy_list/policy_table/policy_table.tsx

```diff
@@ -199,99 +199,93 @@ export const PolicyTable: React.FunctionComponent<Props> = ({
           render: ({ name, inProgress, isManagedPolicy }: SlmPolicy) => {
             return (
               <EuiFlexGroup gutterSize="s">
-                <EuiFlexItem>
```

Comment:
I see there's still the outer `<EuiFlexGroup>` on L201, so I think we should keep `EuiFlexItem` to follow the [EUI usage examples](https://elastic.github.io/eui/#/layout/flex), but `grow={false}` needs to be set for each one.